core/domain/exp_jobs_one_off.py

```diff
@@ -33,14 +33,16 @@
 from core.domain import rights_manager
 from core.platform import models
 import feconf
+from pylatexenc import latex2text
 import python_utils
+import schema_utils
 import utils
 
 (exp_models,) = models.Registry.import_models([
     models.NAMES.exploration])
 gae_image_services = models.Registry.import_gae_image_services()
 
-SEPARATORS = ['<', '>', '=']
+MATH_VALID_EXP_LIMIT = 200
```

Comment:
I don't know what this means. Please pick a clearer name.

Reply:
I've updated the var name and added a comment.